app/src/main/java/org/mozilla/fenix/home/blocklist/BlocklistHandler.kt

```diff
@@ -52,6 +53,14 @@ class BlocklistHandler(private val settings: Settings) {
             }
         }
 
+    /**
+     * Filter a list of recent tabs from sponsored urls.
+     */
+    @JvmName("filterContileRecentTab")
+    fun List<RecentTab>.filterContile(): List<RecentTab> = filterNot {
```

Comment:
Looking just at the parent class, it's documentation and current methods adding here the contile filtering is a bit unexpected - it only referred to a list of websites blocked by the user and persisted locally.
Agree that applying filters in cascade seems to match the current arch though and having them in one place does help. Maybe just the class documentation can be updated to match it's functionality expansion?